docker/files/run/run.sh

```diff
@@ -41,40 +72,28 @@ function apply_configuration() {
   else
     diagcfg telemetry disable
   fi
-}
 
-function catchup() {
-  local FAST_CATCHUP_URL="https://algorand-catchpoints.s3.us-east-2.amazonaws.com/channel/CHANNEL/latest.catchpoint"
-  local CATCHPOINT=$(curl -s ${FAST_CATCHUP_URL/CHANNEL/$NETWORK})
-  if [[ "$(echo $CATCHPOINT | wc -l | tr -d ' ')" != "1" ]]; then
-    echo "Problem starting fast catchup."
-    exit 1
+  # start kmd
+  if [ "$START_KMD" = "1" ]; then
+    local KMD_DIR="kmd-v0.5"
+    # on intial bootstrap, this directory won't exist.
+    mkdir -p "$KMD_DIR"
```

Comment:
Noticed the config file was written to the wrong directory:
```suggestion
    mkdir -p "$KMD_DIR"
    cd "$KMD_DIR"
```